tests/integration/fundMaliciousToken.test.js

```diff
@@ -143,6 +142,8 @@ describe('fund-malicious-token', () => {
       .requestInvestment(dummyAmount, dummyAmount, maliciousToken.options.address)
       .send({ ...investorTxOpts, value: amguAmount });
 
+    await increaseTime(5); // to avoid executing in same block as update
```

Comment:
This is fine here, but just FYI, I'm introducing a simple `delay(ms)` helper in `fix/deploy-testnet` that forces a real pause, so I will refactor this to use that helper once merged. This will allow our tests that only need to increment time by a small amount (several seconds) to work on testnets, since they don't rely on evm manipulation.